pkg/api/elements.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"context"
+	"github.com/permitio/permit-golang/pkg/config"
+	"github.com/permitio/permit-golang/pkg/errors"
+	"github.com/permitio/permit-golang/pkg/models"
+	"github.com/permitio/permit-golang/pkg/openapi"
+	"go.uber.org/zap"
+)
+
+type Elements struct {
+	permitBaseApi
+}
+
+func NewElementsApi(client *openapi.APIClient, config *config.PermitConfig) *Elements {
+	return &Elements{
+		permitBaseApi{
+			client: client,
+			config: config,
+			logger: config.Logger,
+		},
+	}
+}
+
+// LoginAs login as a given user.
+// Usage Example:
+// ```
+// userLogin := models.NewUserLoginRequestInput("user-id", "tenant-id")
+// embeddedLoginRequestOutput, err := PermitClient.Api.Elements.LoginAs(ctx, userLogin)
+// ```
+func (e *Elements) LoginAs(ctx context.Context, userLogin models.UserLoginRequestInput) (*models.EmbeddedLoginRequestOutput, error) {
+	err := e.lazyLoadContext(ctx)
```

Comment:
Maybe just call it once at the api client init, instead of calling it from everywhere?

Reply:
This is a function for the users - like get user, they will need also to loginAs - it's an api call for our new Elements feature.